scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2334,6 +2591,7 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
         next()
         cases += caseClause()
       }
+      if (token.is[LF] && ahead(token.is[CaseIntro])) accept[LF]
```

Comment:
why is this needed? We are already inside cases, so everything should work as previously.

Reply:
okay, so parsing seq of stats is based on a loop: 
`while(CanStartStat) { parseStat; parseSep }`
match-case old version always ends with `}` + LF
match-case new version might only end with Outdent.
I add LF artificially (you can see `insertOutdent()` + insertLF in code) to meet previous assumptions.

But I tried remove this line, remove adding LF after Outdent and add checking in `parseSep` if maybe previous token was Outdent, but I failed and in so many weird places I dropped this idea.

Reply:
Tried to work around it to see if we will get a bigger problem and I think this is the best we can do. Let's leave it.